src/validation_workflow/yum/validation_yum.py

```diff
@@ -69,6 +68,7 @@ def cleanup(self) -> bool:
         try:
             for project in self.args.projects:
                 execute(f'sudo systemctl stop {project}', ".")
+                execute(f'sudo yum remove {project} -y', ".")
```

Comment:
Not really keen on the idea to run everything in `.`, I would rather we have some fixed working dir, such as the root of the build repo.

cc: @gaiksaya thoughts?

Reply:
Since this is for validation only I am okay to remove everything installed as a part of the package. Do you have any concerns regarding it?

Reply:
Hi @gaiksaya, I have addressed the issue to remove all opensearch installations before ending the workflow. I think Peter is talking about "." given as current working directory parameter to execute module instead of a specific path which appears in the logging statements for example `Executing "sudo systemctl stop opensearch" in .` in the output log. Not sure of the path where I need to run all the commands.